include/nlohmann/detail/input/input_adapters.hpp

```diff
@@ -446,6 +446,10 @@ template < typename CharT,
                int >::type = 0 >
 contiguous_bytes_input_adapter input_adapter(CharT b)
 {
+    if(b == nullptr)
+    {
+        JSON_THROW(std::invalid_argument("cannot parse a null string"));
```

Comment:
Please throw `json::parse_error`. Please check `docs/mkdocs/docs/home/exceptions.md` for an overview.